src/components/ListItem.js

```diff
@@ -31,6 +32,17 @@ const ListItem = ({ itemData }) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [itemData]);
 
+
+  //to delete items
+ const deleteItem = async() =>{
+     if(window.confirm("Are you sure you want to delete this item?")){
+        await deleteDoc(doc(db, "Lists", itemData.id));
```

Comment:
"Lists" string contains more than 2. It will be better if we move this to a constant variable.

Reply:
Maybe we can put the whole document reference in a constant variable? and as we use it inside the `handleChnage` function too we can create just one variable at the top of our component and then use it in both functions.
`const docRef = doc(db, "Lists", itemData.id)`